x/service/keeper/relay_mining_difficulty.go

```diff
@@ -29,11 +29,9 @@ func (k Keeper) GetRelayMiningDifficulty(
 	storeAdapter := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
 	store := prefix.NewStore(storeAdapter, types.KeyPrefix(types.RelayMiningDifficultyKeyPrefix))
 
-	difficultyBz := store.Get(types.RelayMiningDifficultyKey(
-		serviceId,
-	))
+	difficultyBz := store.Get(types.RelayMiningDifficultyKey(serviceId))
 	if difficultyBz == nil {
-		difficulty.ServiceId = serviceId
+		difficulty = NewDefaultRelayMiningDifficulty(ctx, k.logger, serviceId, TargetNumRelays)
```

Comment:
Can you just log that nothing exists so we're returning the default w/ TargetnumRelays? Could be a warning.